### PR TITLE
[FIX] Wrong Total on pos order partial return

### PR DESCRIPTION
**Pull request by ajayjaviya**
Task : https://www.druidoo.io/web#id=2622&action=367&model=project.task&view_type=form&menu_id=237

Fix Partail return on pos order is not working when l10n_fr_pos_cert module is installed

https://www.druidoo.io/web#id=2589&action=367&model=project.task&view_type=form&menu_id=237

### CODE REVIEW

**Comment by ivantodorovich** [comment]
Don't forget to add druidoo in contributors.md file ;)

File: pos_order_return/models/pos_order.py [hunk 1]
```diff
@@ -87,9 +86,10 @@ def partial_refund(self, partial_return_wizard):
...
             if qty != 0:
-                copy_line = wizard_line.pos_order_line_id.copy()
+                copy_line = wizard_line.pos_order_line_id.copy(
+                    {'order_id': new_order.id}
+                )
                 copy_line.write({
-                    'order_id': new_order.id,
                     'returned_line_id': wizard_line.pos_order_line_id.id,
                     'qty': qty,
                 })
```
Comment (ivantodorovich) [inline pos_order_return/models/pos_order.py]: Doesns't it make sense to add all the fields in `copy()`, and avoid calling `write()` after?

**Review by ivantodorovich — CHANGES_REQUESTED** [review]

**Comment by ajayjaviya** [comment]
> Don't forget to add druidoo in contributors.md file ;)

@ivantodorovich : done

File: pos_order_return/models/pos_order.py (hunk 1 above)
Comment (ajayjaviya) [reply pos_order_return/models/pos_order.py]: yes , it make sense i have move all field to copy

File: pos_order_return/README.rst [hunk 2]
```diff
@@ -111,6 +111,7 @@ Contributors
 * Sylvain LE GAL <https://twitter.com/legalsylvain>
 * David Vidal <david.vidal@tecnativa.com>
 * Kiril Vangelovski <kiril@lambda-is.com>
+* Druidoo <https://www.druidoo.io>
```
Comment (ivantodorovich) [inline pos_order_return/README.rst]: This file is going to be replaced with the  contents of `readme/CONTRIBUTORS.md`. That's where you need to add Druidoo, not here

**Review by ivantodorovich — CHANGES_REQUESTED** [review]

**Comment by ivantodorovich** [comment]
Please clean you comits after:
it has to be only 2 commits: one for each change, clean messages.

**Comment by ivantodorovich** [comment]
after you applied these changes, you can close this PR and create a PR to OCA. Ask me to review there

**Pull request closed without merging**